tictactoe-domain/src/test/java/camp/nextstep/edu/tictactoe/domain/TictactoeGameTest.kt

```diff
@@ -0,0 +1,110 @@
+package camp.nextstep.edu.tictactoe.domain
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Assert.assertThrows
+import org.junit.Before
+import org.junit.Test
+
+class TictactoeGameTest {
+    private lateinit var game: TictactoeGame
+
+    @Before
+    fun setUp() {
+        game = TictactoeGame()
+    }
+
+    @Test
+    fun `입력하고 같은자리에 입력하면 에러가 발생한다`() {
+        // given
+        game.setPosition(CellPosition.TOP_LEFT)
+
+        // then 에러발생
+        assertThrows(IllegalStateException::class.java) {
+            // when 같은 자리에 입력
+            game.setPosition(CellPosition.TOP_LEFT)
+        }
+    }
+
+    @Test
+    fun `X 3줄이 완성 되면 게임이 종료`() {
```

Comment:
```suggestion
    fun `X 3줄이 완성 되면, X의 승리로 게임이 종료`() {
```
조금더 구체적인 테스트케이스명을 작성하면 더 좋을거 같아요 😄

Reply:
수정적용해 보았습니다. 디테일한 부분까지 리뷰 감사합니다~